discovery/kubernetes/node.go

```diff
@@ -210,31 +211,20 @@ func (n *Node) buildNode(node *apiv1.Node) *targetgroup.Group {
 // 4. NodeExternalDNS
 // 5. NodeLegacyHostIP
 // 6. NodeHostName
+// you can redefine the priority using "preferred_node_address_types" in configuration
```

Comment:
The above comment is no longer correct.

What about:

```
nodeAddresses returns the provided node's address, based on the node 
preferredAddressTypes
```

Reply:
No problem, this will be much simpler than the current comments